### PR TITLE
Add style to exception page

### LINKED ISSUE #7047: Show 4XX errors in theme template
Part of #2800

Currently, 404 errors are shown in a black on white page with no navigation. This is a problem for iOS "Add to Home Screen" as there aren't any navigation element provided by the phone itself. And with the recent change where the app isn't refreshed any time it's opened, the app can stay stuck on the barren 404 page for a while.

Unlike critical 500 errors, 4XX errors are manageable on Friendica side and can be rendered in the current theme template including the navigation.

### PR DESCRIPTION
**Pull request by MrPetovan**
Closes #7047

Thanks to @lostinlight for their contribution. [Their proposition](https://gitlab.com/lostinlight/per_aspera_ad_astra/tree/master/friendica-404) was way more ambitious than I can make or that Friendica themes can take, but I'm eternally grateful anyway because it looks so much better already.

## Duepuntozero
friendica-not-found-duepuntozero
## Frio
friendica-not-found-frio
## Quattro
friendica-not-found-quattro
## Smoothly
friendica-not-found-smoothly
## Vier
friendica-not-found-vier

Additional small fixes:
- On frio, the icon mask svg tag was taking up one text line even though it wasn't displayed.
- There was a typo in the template variable names passed to the `exception.tpl` template preventing the message to display below the title.

### CODE REVIEW

File: view/global.css [hunk 1]
```diff
@@ -621,3 +620,22 @@ span.emoji.mastodon img {
...
+
+#exception .hare {
+  float: right;
+}
+
+@media screen and (max-width: 600px) {
+  #exception .hare {
+    display: none;
+  }
+}
```
Comment (nupplaphil) [inline view/global.css]: Code Standard: EOF

**Comment by nupplaphil** [comment]
Looks great! :-)

**Comment by annando** [comment]
How does it look on mobile?

**Comment by MrPetovan** [comment]
> How does it look on mobile?

On vertical screen in frio, the hare is hidden to leave horizontal room. I didn't check other themes because they don't have explicit mobile capabilities.

**Comment by nupplaphil** [comment]
Looks nice :-) already tested on my dev-node

**Comment by hoergen** [comment]
Do I have to do something to get those screens? If I use an invalid display variable e.g. https://horche.demkontinuum.de/display/52acf9cdee232e420142e3fec7ecaf5293f40c2342 I only get the message "not found". But no graphics.

**Comment by nupplaphil** [comment]
It should look like:
https://friendica.philipp.info/wrongPath
But you're right, I don't find a way to show it with your page

**Comment by hoergen** [comment]
Hmm when I search for the image name (on your page) in my directory, I can't find it. 
Is that part already in "develop" branch?

**Comment by hoergen** [comment]
Ok somebody did something while i was writing. Now it is in develop and vanished in master ;)

**Comment by hoergen** [comment]
It works now beautifully. Thank you 🌻
https://horche.demkontinuum.de/pfrongp%C3%A4f

**Pull request merged**